dodola/core.py

```diff
@@ -357,6 +357,12 @@ def standardize_gcm(ds, leapday_removal=True):
         coords_to_drop, drop=True
     )
 
+    # Some models have coordinates values (e.g. [1.0, 2.0]) for the spatial bounds dimension. We don't need this.
+    if "bnds" in ds_cleaned.variables:
+        ds_cleaned = ds_cleaned.drop(
```

Comment:
...Interesting because xarray's error message says to use `.drop()`:

> ValueError: when setting `region` explicitly in to_zarr(), all variables in the dataset to write must have at least one dimension in common with the region's dimensions ['time'], but that is not the case for some variables here. To drop these variables from this dataset before exporting to zarr, write: .drop(['bnds'])

Might be worth a PR to xarray updating this message.